src/components/products/umi/index.js

```diff
@@ -65,13 +65,7 @@ export const umi = {
         {
           title: 'Guides',
           links: [
-            { title: 'What is Umi?', href: '/umi/guides/what-is-umi' },
-            { title: 'RPCs and DAS', href: '/umi/guides/rpcs-and-das' },
-            { title: 'Umi Programs', href: '/umi/guides/umi-programs' },
-            {
-              title: 'Understanding PDAs',
-              href: '/umi/guides/understanding-pdas',
-            },
+            { title: 'Watch this space!', href: '' },
```

Comment:
What does this mean?

Reply:
just a blank link in the menu, it saves me having to delete the whole page then put it back as I believe we have some Umi stuff coming very shortly.

Reply:
https://developer-hub-git-tony-sem-rush-fix-0d4725-metaplex-foundation.vercel.app/umi/guides